packages/scrapers-v2/src/urls/types.ts

```diff
@@ -0,0 +1,21 @@
+export enum College {
+  ARTS_MEDIA_DESIGN = "arts-media-design",
+  BUSINESS = "business",
+  KHOURY = "computer-information-science",
+  ENGINEERING = "engineering",
+  HEALTH_SCIENCES = "health-sciences",
+  SCIENCE = "science",
+  SOCIAL_SCIENCES_HUMANITIES = "social-sciences-humanities",
+}
+
+export type CatalogHierarchy = {
+  [part: string]: string | CatalogHierarchy;
+};
+
+export type CatalogPath = {
```

Comment:
any reason we don't just do? 
```
export type CatalogPath = string[]
```

Reply:
ya that works too then we don't even need the type `CatalogPath`. Keeping it this way so that people know what the array means haha

Reply:
yea 100% agree w having the type instead of just string[] but perhaps the object nesting isn't required. all good if it's too much work to change tho

Reply:
the reason it's an object is because we originally had a `baseUrl` field in there as well, and then we removed it later. ig an argument could be made for keeping it object in case we want to add a field later, but ya i think its fine to change for now

Reply:
actually imma leave it as is because im lazy